### DISCUSSION TITLE
Possible candidates

### DISCUSSION THREAD

**Discussion by RunningWithTurtles** [aggregatedfinance/agfi-dao#2] [Q&A]
Is there a list of candidates to choose from?

**Comment by cl-tim** [comment]
We're expecting a couple of the project team members to put themselves forward, but it's really open to anyone if they're willing to volunteer to take on these duties for the community.

I should add it's not like it's something that requires multiple hours a day to manage or anything - DAOs never move that fast - it's just a matter of checking in every few days or so and helping evolve the standards of what criteria is essential for a proposal to move to the chain.
